.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -62,7 +62,7 @@ ENV HOME=/home
 ENV OS_ARCHITECTURE="linux-x86_64"
 
 # install OS tools
-RUN apk update && apk add unzip curl rsync openjdk11 jq bash nodejs npm git
+RUN apk update && apk add unzip curl rsync openjdk11 jq bash nodejs npm git uuidgen
```

Comment:
Is there anyway to generate a random String without a new dependency? Every new dependency is subject to vulnerabilities.

Reply:
Took sometime but I found `$RANDOM` should be sufficient.